test/e2e/cluster_config.sh

```diff
@@ -35,7 +35,7 @@ clusters:
     nlb_switch: "pre"
     vm_dirty_bytes: 134217728
     vm_dirty_background_bytes: 67108864
-    prometheus_tsdb_retention_size: enabled
+    prometheus_retention_size: 1280MB
```

Comment:
This is the value that would have been used for test clusters, the only user of this config item (to be exact: 1258292KB)

Reply:
I think this is used for e2e test clusters, the actual value probably does not make any difference as these clusters are short-lived